### ISSUE TITLE
Coveralls button displaying under Bugs category

### ISSUE THREAD

**Issue by Choonster** [apxltd/what-bugs#163]
OS: Windows 10
Browser: Chrome 60.0.3112.113

On the Meta category page, a Coveralls coverage button is displaying under the Bugs category.

![wtdwtf_bugscategory_coverallsbutton_2017-09-07](https://user-images.githubusercontent.com/878073/30148984-34695eac-93e9-11e7-9714-3279ece72708.PNG)

**Comment by boomzillawtf** [comment]
Yes, that was deliberately put there in the category description.

**Comment by pauljherring** [comment]
Is it meant to be squashed up in a 46px<sup>2</sup> like that though? Rather than, say:

![image](https://user-images.githubusercontent.com/3939886/30159011-13f5a790-93bf-11e7-9b41-34af95a8d7ea.png)

Or is that part of the joke?

**Comment by boomzillawtf** [comment]
It looks like the result of matching this CSS selector: `.category>ul>li .content img`, which gives the 46x46 size. Which I believe was meant to match the subcategory icons.
